ntbs-service/Helpers/NotificationHelper.cs

```diff
@@ -43,5 +43,10 @@ public static void SetStartingEventDate(TreatmentEvent treatmentStartEvent, Clin
                 ? TreatmentEventType.TreatmentStart
                 : TreatmentEventType.DiagnosisMade;
         }
+
+        public static string FormatNhsNumber(string rawNhsNumber)
+        {
+            return rawNhsNumber.Replace(" ", "");
+        }

```

Comment:
Given that this is just removing spaces, would a name like `RemoveSpaces` be better, instead of tying this to NHS Numbers when we might want to use this for other things in the future? What do you think?

Reply:
I think `FormatNhsNumber` better captures the encapsulation we're making here, which is specifically the formatting of NHS number strings. If we wanted to remove spaces from other strings, then we could just call `.Replace` directly, but this method allows us to future-proof the logic around formatting NHS numbers so that if it changes it will only need to be updated in a single place.